apps/site/lib/site_web/templates/trip_plan/_fare_calculator.html.eex

```diff
@@ -0,0 +1,61 @@
+
+<div class="h2 m-trip-plan-farecalc-title">Fare Calculator</div>
```

Comment:
I think some of these class names could have the "element" portion separated out per our [BEM](http://getbem.com/naming/) standard. I.e. maybe this would be `m-trip-plan-farecalc__title`, and a couple examples from below would be `m-trip-plan-farecalc__table` and `m-trip-plan-farecalc__table-cell`.

Reply:
Those examples are great. These conventions are also described [here](https://github.com/mbta/wiki/blob/4185fb315937c1370374a88f0555c32b6d44d090/website/development/guidelines/css_guidelines.md).